cns/NetworkContainerContract.go

```diff
@@ -233,8 +233,8 @@ type GetIPAddressesRequest struct {
 
 // GetIPAddressStateResponse is used in CNS IPAM mode as a response to get IP address state
 type GetIPAddressStateResponse struct {
```

Comment:
I would leave it as is, and expose a new struct named GetIPAddressStatusResponse

Reply:
Ack, Addressed in the recent push @neaggarwMS